### PR TITLE
[incubator/couchdb] typo fix in README.md  

### PR DESCRIPTION
**Pull request by AdamDang**
Two typos:
1) Some proper noun in capital, some not. fix to be in capital letters.
2) tables lists->tables list

### CODE REVIEW

File: incubator/couchdb/README.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ deletes the release.
 
 ## Configuration
 
-The following tables lists the most commonly configured parameters of the
+The following tables list the most commonly configured parameters of the
```
Comment (rjkernick) [inline incubator/couchdb/README.md]: Shouldn't it be "table lists"

Comment (AdamDang) [reply incubator/couchdb/README.md]: OK, I've correct it.

**Pull request merged**